ncbi_genome_download/core.py

```diff
@@ -280,17 +294,17 @@ def _download(section, group, uri, output, file_format, assembly_level, genus, s
             logging.debug('Skipping entry with assembly level %r', entry['assembly_level'])
             continue
         download_jobs.extend(
-            download_entry(entry, section, group, output, file_format, human_readable))
+            download_entry(entry, section, group, output, file_format, human_readable, proxies))
     return download_jobs
 # pylint: enable=too-many-arguments
 
 
-def worker(job):
+def worker(job, proxies=None):
```

Comment:
Actually, the job is a `namedtuple('DownloadJob', ['full_url', 'local_file', 'expected_checksum', 'symlink_path'])` which does not contain the proxies. This is the reason why the proxies are an additional parameter to `worker()` and thus the `star_map` trick - which I also do not like much.
The reason behind this situation is that I didn't wanted to propagate the proxies too far, as they are neither used in `download_file_job` nor in `create_symlink_job`, where they would just be added to the `DownloadJob` namedtuple.